index.bs

```diff
@@ -1248,7 +1248,7 @@ partial interface MLGraphBuilder {
         - *pow*: Compute the values of the values of the first input tensor to the power of the values of the second input tensor, element-wise.
 </div>
 
-### element-wise unary operations ### {#api-mlgraphbuilder-unary}
+### Element-wise unary operations ### {#api-mlgraphbuilder-unary}
```

Comment:
ditto.

Reply:
Same argument as above. I suggest either keeping the generic section name, or split up to subsections for each method.